plugins/influxdb/influx.go

```diff
@@ -0,0 +1,26 @@
+package influxdb
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func InfluxDBCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "InfluxDB CLI",
+		Runs:    []string{"influx"},
+		DocsURL: sdk.URL("https://docs.influxdata.com/influxdb/"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
+			needsauth.NotWhenContainsArgs("config"),
```

Comment:
Let's see authentication for `version` and `completion`. Commands like `backup` and `restore` are specific to InfluxDB OSS and InfluxDB Cloud, but I am still figuring out whether we should exclude them entirely or build support for InfluxDB OSS too. I'll followup here.

Reply:
I have not done any changes to this comment. My assumption we are supporting only influxdb cloud at this moment via shell plugin